modules/gdscript/gdscript.cpp

```diff
@@ -915,14 +915,43 @@ GDScript::GDScript() :
 #endif
 }
 
+void GDScript::_orphan_subclasses() {
```

Comment:
I think this name can be better, it's not clear what it does. Maybe `_save_orphaned_subclasses()`?